asm/internal/ll.bnf

```diff
@@ -1719,8 +1726,7 @@ OptAllocSizeNElems
 ;
 
 Elems
-	: empty
-	| ElemList
+	: ElemList
```

Comment:
Why did you remove `empty`? The convention used to this point is that `Foos` represent 0 or more `Foo`s and `FooList` represents 1 or more `Foo`s.